Mediafiles get duplicated in mediaservice with organization.disable_forward_with_attachments

Occurs on attempt to forward motion in the organization with the setting `organization.disable_forward_with_attachments`.

**How to reproduce:**
In the end of the test `test_forward_with_attachment_true_and_forward_with_attachments_disabled` add check `self.media.duplicate_mediafile.assert_not_called()`.

**Current behaviour:**
```
AssertionError: Expected 'duplicate_mediafile' to not have been called. Called 2 times.
Calls: [call(3, 8), call(4, 9)].
```
Mediafiles don't get forwarded in the database but meeting-wide mediafiles get duplicated in the mediaservice.

**Expected behaviour:**
When mediafiles can not get forwarded, they should not be duplicated.

--- a/openslides_backend/action/actions/motion/base_create_forwarded.py
+++ b/openslides_backend/action/actions/motion/base_create_forwarded.py
@@ -458,25 +458,29 @@ class BaseMotionCreateForwarded(
         meeting_mediafile_replace_map: dict[int, dict[int, int]],
     ) -> tuple[dict[int, set[int]], dict[int, dict[int, int]]]:
         # Extract mediafiles and meeting_mediafiles data
-        motion_target_meeting_ids_map: dict[int, set[int]] = (
-            self._extract_motion_target_meeting_ids(action_data)
-        )
-        origin_attachments_data: dict[int, dict[str, Any]] = (
-            self._fetch_origin_attachments_data(
-                list(motion_target_meeting_ids_map.keys())
+        if not self.datastore.get(
+            ONE_ORGANIZATION_FQID, ["disable_forward_with_attachments"]
+        ).get("disable_forward_with_attachments"):
+            motion_target_meeting_ids_map: dict[int, set[int]] = (
+                self._extract_motion_target_meeting_ids(action_data)
             )
-        )
-        fetched_data = self._prepare_mediafiles_data(
-            motion_target_meeting_ids_map,
-            origin_attachments_data,
-            forwarded_attachments,
-        )
+            origin_attachments_data: dict[int, dict[str, Any]] = (
+                self._fetch_origin_attachments_data(
+                    list(motion_target_meeting_ids_map.keys())
+                )
+            )
+            fetched_data = self._prepare_mediafiles_data(
+                motion_target_meeting_ids_map,
+                origin_attachments_data,
+                forwarded_attachments,
+            )
 
-        # Calculate new ids and execute dublication actions
-        meeting_mediafile_replace_map = self.perform_mediafiles_duplication(
-            fetched_data, meeting_mediafile_replace_map
-        )
-        return forwarded_attachments, meeting_mediafile_replace_map
+            # Calculate new ids and execute dublication actions
+            meeting_mediafile_replace_map = self.perform_mediafiles_duplication(
+                fetched_data, meeting_mediafile_replace_map
+            )
+            return forwarded_attachments, meeting_mediafile_replace_map
+        return {}, {}
 
     def _extract_motion_target_meeting_ids(
         self, action_data: ActionData

--- a/tests/system/action/motion/test_create_forwarded.py
+++ b/tests/system/action/motion/test_create_forwarded.py
@@ -1707,6 +1707,7 @@ class CreateForwardedTestWithAttachmentsSimple(
         assert response.json["message"] == "Forward with attachments is disabled"
         self.assert_model_not_exists("mediafile/7")
         self.assert_model_not_exists("meeting_mediafile/17")
+        self.media.duplicate_mediafile.assert_not_called()
 
     def test_forward_to_the_same_meeting_with_orga_wide_mediafile(self) -> None:
         """

